docs/examples/checking_and_inference/order.txt

```diff
@@ -0,0 +1,7 @@
+1. arith
```

Comment:
it's the order in which i would show Edwards the examples - this is just a rough draft, liable to change + input from you guys. It sounds like I may have made too many examples, and may need to condense into only one or two.

@akalicki